### DISCUSSION TITLE
Recommendations for Smartphone use

### DISCUSSION THREAD

**Discussion by hgj44d** [HotCakeX/Harden-Windows-Security#228] [Q&A] [answered]
Hi!

I think most people don't use Windows only, but also mobile devices with a smartphone OS like Android or iOS. What are recommended ways for using a smartphone in conjunction with Windows? I know this question sounds highly off-topic, but I think the choices which you do on Windows also effects the use on a smartphone. I have divided my questions into the following areas:

### Hardware

First topic would be which smartphones would be considered as secure. Regarding to Daniel Micay the following smartphones are considered as secure:

- Google Pixel devices
- Apple iPhone devices
- Samsung devices ("[Samsung takes security almost as seriously as Google](https://www.kuketz-blog.de/weshalb-grapheneos-aktuell-nur-google-pixel-geraete-unterstuetzt/)" and [here](https://discuss.grapheneos.org/d/9882-is-samsung-knox-better/4))

Any opinions about that?

### Password manager

Regarding to the information in this repository and various other sources the most secure browser on Windows is Microsoft Edge. The most secure password manger (in general) seems to be the one build into the browser, because all other methods would require a browser extension which is considered as insecure. This means you should use the build-in Edge password manager. How do you recommend using these passwords on Android/iOS? Also using the Edge browser on Android/iOS? Or sync the passwords with the Authenticator app and then use the AutoFill feature of Android/iOS to get the passwords on the default browser (Chrome, Safari, Samsung Internet Browser)?

### Cloud storage

I guess the most secure cloud storage on Windows is OneDrive (I found OneDrive mentioned in some policies, so this is only a assumption)? 

In the past I tried setting up OneDrive on a Google Pixel and the synchronisation with images was not working correctly: if I deleted an image in the Google Photos app, the image was still in OneDrive. No real two-way sync. I didn't tested it yet again, but I think it's still an issue. Can anybody share their experience with Google Pixel devices and OneDrive?

It shouldn't be an issue on Samsung devices if I read that correctly. Samsung seems to work heavily together with Microsoft recently and regarding to their website OneDrive is the default cloud storage and stuff like two-way sync of images with OneDrive should work. Can anybody share their experience with Samsung devices and OneDrive?

What if I use Google Drive on Windows and Android/iOS, are there any concerns?

### Link to Windows and Microsoft Phone Link

Is [Microsoft Phone Link](https://support.microsoft.com/en-US/phone-link) considered as secure or should I avoid it?

**Comment by HotCakeX** [comment, marked as answer]
Hi, this is a great post, thanks for taking the time and compiling it.

I think in terms of security, Google Pixel phones with GrapheneOS is a good choice. Sadly they don't support any other types of phones.

For password manager, I use Microsoft Authenticator device-wide, that's what businesses and enterprises use as well for mobile device management etc. It's as secure as any other password manager if not more.

There is no security reason to avoid using Microsoft's Phone Link. I use it on my phone too.

**Reply by hgj44d** [reply]
Thanks for answering!

What are your thoughts on OneDrive vs. Google Drive?

**Reply by HotCakeX** [reply]
My pleasure,

I think OneDrive is better, that's what I use, all my data remains in one company, comes preinstalled with Windows so it's not a 3rd party program and integrates well with all the programs, file explorer, M365 apps etc.
